test/facade_converter_test.ts

```diff
@@ -73,6 +73,10 @@ describe('type based translation', () => {
       expectWithTypes('var n: Node;').to.equal(' dynamic n ;');
       expectWithTypes('var _xhr: XMLHttpRequest;')
           .to.equal(' import "dart:html"; HttpRequest _xhr ;');
+      expectWithTypes('var _intArray: Uint8Array;')
```

Comment:
could you remove the leading underscores from the variables? Please also remove them on the line above.